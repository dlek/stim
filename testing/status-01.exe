#!/bin/bash
#
#
TEST_SCRIPT=$(basename $0)
TEST_NAME=${TEST_SCRIPT%*.exe}
TEST_DESCRIPTION="Test basic status"
TEST_HOME=$(dirname $0)
TEST_BASE=${0%*.exe}
TEST_EXPECTED=${TEST_BASE}.expected

export STIM_HOME=${TEST_HOME}
export STIM_CONTRACT=${TEST_NAME}

export STIM_FAKE_TIME=1100591972

if TEST_DIFF=$($STIM status | diff - ${TEST_EXPECTED})
then
  success
else
  failed
fi

