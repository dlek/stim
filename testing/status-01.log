20041115 10:25:00 start General/Meetings
20041115 11:05:00 start General/Communication
20041115 11:25:00 start Project 1/Maintenance
20041115 11:38:48 start Project 2/Task X
20041115 12:32:28 start Project 1/Development
20041115 13:01:07 stop
20041115 13:21:32 start Project 1/Maintenance
20041115 14:08:47 start Project 1/Development
20041115 16:49:47 stop
20041115 18:18:51 start Project 1/Development
20041115 18:54:43 start Project 2/Task X
20041115 20:39:57 stop
20041115 21:40:46 start Project 2/Task X
20041115 22:52:29 stop
20041115 23:56:36 start Project 2/Task X
