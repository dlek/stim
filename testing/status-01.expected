Session time: 09:40:07
Task time:    03:53:33
Task:         Project 2/Task X
Timer is      running
